packages/aws-cdk/lib/api/evaluate-cloudformation-template.ts

```diff
@@ -34,7 +35,57 @@ export class LazyListStackResources implements ListStackResources {
   }
 }
 
-export class CfnEvaluationException extends Error {}
+export interface LookupExport {
+  lookupExport(name: string): Promise<AWS.CloudFormation.Export | undefined>;
+}
+
+export class LookupExportError extends Error { }
+
+export class LazyLookupExport implements LookupExport {
+  private cachedExports: { [name: string]: AWS.CloudFormation.Export } = {}
+
+  constructor(private readonly sdk: ISDK) { }
+
+  async lookupExport(name: string): Promise<AWS.CloudFormation.Export | undefined> {
+    if (this.cachedExports[name]) {
+      return this.cachedExports[name];
+    }
+
+    for await (const cfnExport of this.listExports()) {
+      if (!cfnExport.Name) {
+        throw new LookupExportError(`Unable to handle CloudFormation Export without Name! ${JSON.stringify(cfnExport)}`);
```

Comment:
Good call, we'll just skip over them. At the time I wanted to initially provide the least opinionated implementation

Reply:
We can skip the [exports](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_Export.html) with not `Name`. It would be strange to see such a scenario, have you encountered these?

Reply:
I have not encountered that scenario, no. I'll update to skip over exports with no `Name`! 🙌